mesh/mesh.cpp

```diff
@@ -8926,11 +8926,11 @@ void Mesh::InitFromNCMesh(const NCMesh &ncmesh)
    // outside after this method.
 }
 
-Mesh::Mesh(const NCMesh &ncmesh)
+Mesh::Mesh(const NCMesh &other_ncmesh)
```

Comment:
I'd prefer `ncmesh_`, which is the convention used elsewhere in constructors (or "constructor-like" functions)

Reply:
Done